docs/large_inputs.md

```diff
@@ -1,63 +1,127 @@
 # Handling large inputs
 
 The Variant Transforms pipeline can process hunderds of thousands of files,
-millions of samples, and billions of records. Please consider adjusting the
-default settings as described below when processing any dataset that is more
-than a few gigabytes or contains more than 50,000 files (even if the individual
-files are small).
+millions of samples, and billions of records. There are a few settings that
+may need to be adjusted depending on the size of the dataset. Each of these
+settings are explained in the sections below.
 
-In addition, it is recommended to run the pipeline with
-`--optimize_for_large_inputs` to process large datasets more efficiently.
+Default settings:
 
-## Numer of workers
+```
+/opt/gcp_variant_transforms/bin/vcf_to_bq ... \
+  --optimize_for_large_inputs <default false> \
+  --max_num_workers <default is automatically determined> \
+  --worker_machine_type <default n1-standard-1> \
+  --disk_size_gb <default 250> \
+  --worker_disk_type <default PD> \
+  --num_bigquery_write_shards <default 1> \
+  --partition_config_path <default None> \
+```
 
-The default maximum number of workers is 15. You can adjust this by adding
-`--max_num_workers <num workers>` to the pipeline args.
-The actual number of workers will be dynamically adjusted based on Dataflow's
-autoscaling algorithm. If you prefer to not use the autoscaling algorithm, you
-may set `--num_workers <num workers>` to use the exact
-number of workers.
+## Important note about adjusting quota
 
-### Adjusting quota
+Compute Engine enforces quota on maximum amount of resources that can be used
+at any time for variety of reasons. Please see
+https://cloud.google.com/compute/quotas for more details. As a result, you may
+need to adjust the quota to satisfy the job requirements.
 
-To increase the number of workers, you need to adjust the Compute Engine quota.
-In particular, you need at least `num_workers + 1` IPs, `num_workers + 1` CPUs,
-and `num_workers * 250GB` of disk in the zone you are running the pipeline.
-Please see https://cloud.google.com/compute/quotas for details on how to
-adjust your quota.
+The main Compute Engine quotas to be adjusted are:
+* `In-use IP addresses`: One per worker.
+* `CPUs`: At least one per worker. More if larger machine type is used.
+* `Persistent Disk Standard (GB)`: At least 250GB per worker. More if larger
+  disk is used.
+* `Persistent Disk SSD (GB)`: Only needed if `--worker_disk_type` is set to SSD.
+  Required quota size is the same as `Persistent Disk Standard`.
 
```

Comment:
It would be nice if we could somehow explain these will be the upper limits of available resources, for example:

> The value assigned to these quotas will be the upper limit of available resources for your job. For example, if you set `In-use IP addresses` to 10 while set `--max_num_workers 20` your job will be running with at most 10 workers because that's all your GCP project is allowed to use.

Reply:
Good point. Updated!